cmake/modules/FindSUNDIALS.cmake

```diff
@@ -58,48 +60,42 @@ DEAL_II_FIND_LIBRARY(SUNDIALS_LIB_SER NAMES sundials_nvecserial
   PATH_SUFFIXES lib${LIB_SUFFIX} lib64 lib
   )
 
-#
-# only define the extra hint if ${SUNDIALS_DIR} is nonempty so that we do not
-# try to search through the top-level '/include/' directory, should it happen to
-# exist
-#
-STRING(COMPARE EQUAL "${SUNDIALS_DIR}" "" _sundials_dir_is_empty)
```

Comment:
this was added by @drwells in https://github.com/juan42/dealii/commit/803546edaadb09e1297076831d709165d07b6263 , these changes probably break something... Let's wait for him to review the PR.